app/recipe/models/recipe.py

```diff
@@ -0,0 +1,10 @@
+from django.db import models
+
+
+class Recipe(models.Model):
```

Comment:
For what is worth, it is not really necessary to create one file per model/class. You can put all model class in `recipe.models.py`. But it is OK to do so if you prefer that :)